doc/api_ref/pubkey.rst

```diff
@@ -1021,6 +1206,11 @@ Parameters specification:
 Ed25519 and Ed448 Variants
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. warning:: Ed25519 and Ed448 have different verification criteria, depending
+   on the implementation. This can be problematic in systems which rely on
+   consensus - see `It’s 255:19AM. Do you know what your validation criteria
+   are? <https://hdevalence.ca/blog/2020-10-04-its-25519am>`_ for details.
+
```

Comment:
For consistency 💪 

```suggestion
.. warning::

   Ed25519 and Ed448 have different verification criteria, depending
   on the implementation. This can be problematic in systems which rely on
   consensus - see `It’s 255:19AM. Do you know what your validation criteria
   are? <https://hdevalence.ca/blog/2020-10-04-its-25519am>`_ for details.

```